File: DMS_Task/ClientApp/src/app/models/item.ts
export class Item {
    constructor(
        public id?: number,
        public name?: string,
        public description?: string,
        public unitOfMeasureId?: number,
        public unitOfMeasure?: string,
        public imageUrl?: string,
        public quantity?:number,
        public avalibleQuantity?: number,
        public unitPrice?: number,
        public discount?: number,
        public tax?:number,
        public limitPerCustomer?: number
    ) { }
}

File: DAL/Models/Item.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DAL.Models
{
    public class Item
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public int UnitOfMeasureId { get; set; }
        public UnitOfMeasure UnitOfMeasure { get; set; }
        public string ImageUrl { get; set; }
        public double Quantity { get; set; }
        public double AvalibleQuantity { get; set; }
        public double UnitPrice { get; set; }
        public double Discount { get; set; }
        public double Tax { get; set; }
        public ICollection<OrderItems> OrderItems { get; set; }
        public bool IsVisible { get; set; }
        public double LimitPerCustomer { get; set; }
    }
}
